src/main/java/uk/nhs/digital/nhsconnect/nhais/model/jsonpatch/AmendmentPatchRemoval.java

```diff
@@ -0,0 +1,27 @@
+package uk.nhs.digital.nhsconnect.nhais.model.jsonpatch;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.ToString;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@EqualsAndHashCode(callSuper = false)
+public class AmendmentPatchRemoval extends AmendmentPatch {
+
+    @NonNull
+    private AmendmentPatchOperation op;
+    @NonNull
+    private String path;
+    @JsonIgnore
+    private AmendmentValue value;
```

Comment:
Should this maybe be something like `final` and `= null`. Or getter and setter throw unsupported operation? It is not possible to use "value" for an `"op": "remove"`